src/renderers/shaders/ShaderChunk/shadowmap_vertex.glsl.js

```diff
@@ -60,4 +45,24 @@ export default /* glsl */`
 	*/
 
 #endif
+
+// spot lights can be evaluated without active shadow mapping (when SpotLight.map is used)
```

Comment:
I hope this comment makes it clear why the below code is executed without a `USE_SHADOWMAP` definition.